packages/flutter/test/widgets/focus_scope_test.dart

```diff
@@ -1006,6 +1006,29 @@ void main() {
       expect(rootNode.hasFocus, isTrue);
       expect(rootNode, equals(firstElement.owner.focusManager.rootScope));
     });
+    testWidgets('Can autofocus a node.', (WidgetTester tester) async {
+      final FocusNode focusNode = FocusNode(debugLabel: 'MaterialButton');
```

Comment:
I don't think the debugLabel makes sense here.

Reply:
Me either!  Thanks, fixed.